### DISCUSSION TITLE
Disable Twitch Experiment

### DISCUSSION THREAD

**Discussion by jazminite** [night/betterttv#7215] [Ideas]
Add a way to disable the Twitch experiment that is putting the title at the top of the video (was at the bottom). FrankerZ app has already implemented this. I'd rather not download another Chrome Extension.

![image](https://github.com/user-attachments/assets/90c67a4f-e303-4431-a123-e2349b7ed9d2)

Reference posts from other people:
https://www.reddit.com/r/Twitch/comments/1ham6am/comment/m19mwup/
https://www.reddit.com/r/Twitch/comments/1hanxs1/how_do_i_fix_the_twitch_title/

**Comment by Steffronte** [comment]
I would also be very interested in this. Was considering FrankerZ too, but it would mean removing BTTV because they are not suppose to work together as far as I know.

**Comment by terryhedge** [comment]
This should definitely be implemented, I hate the look of the title on top of the player...

**Comment by rspurrell** [comment]
Came here to log this very idea.

**Comment by MoDog0** [comment]
Came looking for this. Yes would greatly appreciate a way to put the channel identification, Title, Game, update, etc. back under the stream.

**Comment by dclstn** [comment]
While we decide on the best way to implement this, I wrote a script that'll revert the layout.

https://gist.github.com/dclstn/eae2aea2e20585763c9ed9dba2b60003

**Comment by dclstn** [comment]
Look like twitch has rolled back this change, feel free to reopen if you're still experiencing this layout
